edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
```

Comment:
I don't know if the most effective use of your time is to do a one-time pass once we've completed the full set of functionality. The main missing block right now is the "end-of-exam" workflow. That effort is what we will be completing this week (hopefully). Then we should have a feature complete implementation and we can all sit around all of the various workflows and user states.